src/renderer/components/MainPage.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <section>
+    <hero title="Welcome on our meeting!" />
+
+    <section class="main-container container">
+      <div class="tile is-ancestor">
+        <randomizer-tile v-bind:participantList="participantList" />
```

Comment:
You can omit the `v-bind` part, same as `v-on` can be replaced by `@`, for example: `@click=""`